en/development/configuration.rst

```diff
@@ -626,7 +626,24 @@ This means for example, that if you need to change your database, you'll just
 need to modify a DATABASE_URL variable on your host configuration without the
 need to change it in your source code.
 
-As you can see in your **app.php**, the following variables are concerned:
+Because CakePHP will automatically prefer a setting exposed as an environment
+variable over its peer setting in ``app.php`` enabling environment variables is
```

Comment:
Somehow I find this more unclear now as it was before.

The list starts with creating a `.env` file out of nowhere, and then later mentions a library whichs usage would make that `.env` file being involved at all, ie the first two steps won't do anything out of the box.

The third step in the list feels kinda misplaced, considering the introduction text of this section, which talks about defining environment variables in cloud environments, it reads as if all three points in this list would apply to a local system, hence making number 3 kinda superflous, respectively more of an "or do this", rather than an "and do this".